warehouse/malware/utils.py

```diff
@@ -27,6 +28,15 @@ def valid_check_types():
     return set([t.value for t in MalwareCheckObjectType])
 
 
+def get_check_fields(check):
```

Comment:
I'm not sure why, but I feel a little apprehensive passing all "non-dunder"/non-routine class attributes to the `MalwareCheck` instantiation above

```python
            request.log.info("Adding new %s to the database." % check_name)
             fields = get_check_fields(check)
             request.db.add(MalwareCheck(**fields))
```

It kinda stinks to hardcode the specific class attributes we're interested in, but could have the benefit of allowing us to implement a `validate_check_definition` or similar utility that could ensure that checks conform over time.

I'm imagining such a utility could ensure that all objects subclassing `MalwareCheckBase` contain the specific attributes and methods the interface defines.

Reply:
This could definitely be used maliciously, but has a future-proof/convenience benefits. I'll do the hard-coding thing.